src/languages/es.js

```diff
@@ -522,6 +522,8 @@ export default {
         tryAgain: 'Intentar otra vez',
         verifyIdentity: 'Verificar identidad',
         genericError: 'Hubo un error al procesar este paso. Inténtalo de nuevo.',
+        cameraRequestTitle: '"New Expensify" Would Like to Access the Camera',
```

Comment:
Use the `cameraPermissionsNotGranted` key instead of this and translations should be couple lines above.